eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -357,55 +357,59 @@
+                    myCss.rel = "stylesheet";
+                    myCss.href = "https://unpkg.com/leaflet@1.3.1/dist/leaflet.css";
+                    myCss.crossorigin = "";
                     win.ezgmaplocationMapsScriptLoaded = true;
+                    doc.body.appendChild(myCss);
                     doc.body.appendChild(myScript);
                 })(window, document)
             }
         </script>
         <script type="text/javascript">
-        (function(win, doc) {
-            var mapView = function (mapId, latitude, longitude) {
-                var mapElt = doc.getElementById("{{ mapId }}"),
-                    startPoint = new google.maps.LatLng(latitude, longitude),
-                    zoom = {{ zoom }},
-                    draggable = {{ draggable }};
-
-                new google.maps.Marker({
-                    map: new google.maps.Map(mapElt, {
-                        center: startPoint,
-                        zoom: zoom,
-                        draggable: draggable,
-                        scrollwheel: {{ scrollWheel }},
-                        mapTypeId: google.maps.MapTypeId.{{ mapType }}
-                    }),
-                    position: startPoint
-                });
-            }
-            if ( win.addEventListener ) {
-                win.addEventListener(
-                    'load',
-                    function () {
-                        mapView("{{ mapId }}", {{ latitude }}, {{ longitude }});
-                    },
-                    false
-                );
-            } else if ( win.attachEvent ) {
-                win.attachEvent(
-                    'onload',
-                    function () {
-                        mapView("{{ mapId }}", {{ latitude }}, {{ longitude }});
-                    }
-                );
-            }
-        })(window, document);
+            (function (win) {
+                var mapView = function (mapId, latitude, longitude) {
+                    var mapConfig = {
+                        dragging: {{ zoom }},
+                        scrollWheelZoom: {{ scrollWheel }}
+                    };
+                    var map = L.map(mapId, mapConfig).setView([latitude, longitude], {{ zoom }});
+
+                    L.tileLayer('http://{s}.tile.osm.org/{z}/{x}/{y}.png', {
+                        attribution: '&copy; <a href="http://osm.org/copyright">OpenStreetMap</a> contributors'
+                    }).addTo(map);
+
+                    L.marker([latitude, longitude]).addTo(map);
+                };
+                if (win.addEventListener) {
+                    win.addEventListener(
+                        'load',
+                        function () {
+                            mapView("{{ mapId }}", {{ latitude }}, {{ longitude }});
+                        },
+                        false
+                    );
+                } else if (win.attachEvent) {
+                    win.attachEvent(
```

Comment:
I think this one can be dropped as we don't support IE10 and older browsers. Please, stick to `addEventListener` method only.